sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -199,6 +199,7 @@ public partial class RequestContext
         public void AddClassifier(int statusCode, bool isError) { }
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
         public static implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
+        public void ReplacePolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.PipelinePolicyReplacement replace) { }
```

Comment:
I would call it SetPolicy. .NET naming conventions for collections are (Add/Set/Remove)

Reply:
Do we really think this list will grow enough to warrant the separate method, and type?

Reply:
I really just threw it out there as a suggestion to consider with the others.  The tradeoff is potentially adding more policies that can be replaced to ClientOptions vs. having one thing that grows without increasing the ClientOptions API surface area.  The other piece to consider is consistency across different approaches to setting - would it make sense to use the same API for library authors and protocol method callers, would we be happy with this on RequestContext, etc?  My instinct is to make things more general to be able to respond to unforseen changes later, but if we really think we won't add more policies, I agree the overhead of extra types would be unnecessary.

Reply:
I think the other consideration is usability and discoverability for users. While I like the push towards generalizing the design, my gut says it won't add enough additional policies to justify the UX downsides.